resources/assets/js/helpers.js

```diff
@@ -405,22 +405,10 @@ $.extend(true, laravelValidation, {
          * @param name
          * @returns {RegExp}
          */
-        regexFromWildcard: function(name) {
-            var nameParts = name.split("[*]");
-            if (nameParts.length === 1) {
-                nameParts.push('');
-            }
-            var regexpParts = nameParts.map(function(currentValue, index) {
-                if (index % 2 === 0) {
-                    currentValue = currentValue + '[';
-                } else {
-                    currentValue = ']' +currentValue;
-                }
```

Comment:
This seems to be the problematic part of the old function. I think it should be replaced with the below, then it will produce the same output as your function:
```js
if (index > 0) {
    currentValue = ']' + currentValue;
}

if (index < (nameParts.length - 1)) {
    currentValue += '[';
}
```

Reply:
This also work, if you prefer this over the rewrite I will push it like yours :)